src/pages/toolkit/form-js/index.hbs

```diff
@@ -0,0 +1,80 @@
+---
+title: "form-js: Form Builder and Viewer"
+layout: base.hbs
+---
+
+
+<!-- subhead -->
+
+<div class="bi-masterhead">
+  <div class="container">
+
+    <p class="h1">
+       form-js
+    </p>
+
+    <p class="lead">Visually edit and embed JSON-based forms.</p>
+
+    <div class="actions">
+      <a href="http://demo.bpmn.io/form" target="_blank" class="btn btn-xlg btn-primary">Try Online</a>
+      <a href="{{relative page.dest "dist/toolkit/form-js/download"}}" class="btn btn-xlg btn-default">Download</a>
+    </div>
+
+    <p class="bi-note">
+      Currently {{ site.formjs.version }}. <a href="https://github.com/bpmn-io/form-js">View on GitHub</a>.
+    </p>
+  </div>
+</div>
+
+<!-- features -->
+
+<div class="bi-features">
+  <div class="container">
+    <h2>Create forms visually</h2>
+    <p class="lead">
+      Assemble your forms using our intuitive <a href="https://github.com/bpmn-io/form-js#builder">form builder</a>.
```

Comment:
Is it rather this URL: https://github.com/bpmn-io/form-js#create-and-edit-a-form? Or will we update them?

Reply:
Good catch. I believe we're better of adding stable in-page links. Created https://github.com/bpmn-io/form-js/pull/64 for `#viewer` and `#builder` to work.